GatherBuddy/FishTimer/FishTimerWindow.cs

```diff
@@ -263,6 +263,10 @@ public override void Draw()
         }
         else
         {
+            if (_spotName == null)
+            {
+                _spotName = GetSpotText(_spot);

```

Comment:
`_spotName ??= GetSpotText(_spot);`

Reply:
Thanks, my C# knowledge is old and I forget these exist.

Reply:
That's why I remarked it instead of just changing it after merging! :)

Reply:
I looked at your other changes too! I didn't know `Min` could take a delegate either.  :)